src/analysis/retail/paladin/protection/modules/Abilities.js

```diff
@@ -87,42 +88,40 @@ class Abilities extends CoreAbilities {
       },
       //COOLDOWNS
       {
-        spell: SPELLS.ARDENT_DEFENDER.id,
-        buffSpellId: SPELLS.ARDENT_DEFENDER.id,
+        spell: TALENTS.ARDENT_DEFENDER_TALENT.id,
+        buffSpellId: TALENTS.ARDENT_DEFENDER_TALENT.id,
+        enabled: combatant.hasTalent(TALENTS.ARDENT_DEFENDER_TALENT),
         category: SPELL_CATEGORY.DEFENSIVE,
-        cooldown: 120 * (combatant.hasTalent(SPELLS.UNBREAKABLE_SPIRIT_TALENT) ? 0.7 : 1),
+        cooldown: 120 * (combatant.hasTalent(TALENTS.UNBREAKABLE_SPIRIT_TALENT) ? 0.7 : 1),
         castEfficiency: {
           suggestion: true,
         },
       },
       {
-        spell: [SPELLS.GUARDIAN_OF_ANCIENT_KINGS.id, SPELLS.GUARDIAN_OF_ANCIENT_KINGS_QUEEN.id],
-        buffSpellId: [
-          SPELLS.GUARDIAN_OF_ANCIENT_KINGS.id,
-          SPELLS.GUARDIAN_OF_ANCIENT_KINGS_QUEEN.id,
-        ],
+        spell: SPELLS.GUARDIAN_OF_ANCIENT_KINGS_QUEEN.id,
+        buffSpellId: SPELLS.GUARDIAN_OF_ANCIENT_KINGS_QUEEN.id,
```

Comment:
did they fix GoAK having a different spell ID if you used the Queen glyph?

should probably be e.g. `[TALENTS.GUARDIAN_OF_ANCIENT_KINGS_TALENT.id, SPELLS.GUARDIAN_OF_ANCIENT_KINGS_QUEEN.id]`

Reply:
I think I updated this correctly this time around, I left the spell.id set to kings, but made the suggested change to the buffspellid and this seems to work.